### ISSUE TITLE
Sound problem

### ISSUE THREAD

**Issue by chenbevis** [FrSkyRC/ETHOS-Feedback-Community#773]
The sound should be played at the same time, instead of waiting for the first one to finish before playing the second one

**Comment by Hobby4life** [comment]
I vote to stay like it is now.

I have setup sequences like this in special functions like:

- SF1 Play track: "Battery"
- SF2 Play value: consumption"
- SF3 Play track: "consumed"

**Comment by chenbevis** [comment]
> 我投票保持現狀。
> 
> 我在特殊功能中設置了這樣的序列，例如：
> 
> * SF1 播放曲目：“電池”
> * SF2 遊玩價值：消費”
> * SF3 播放曲目：“已消耗”
> 
> 現在它以正確的“階梯”順序被調用。

If you are playing music and the telemetry alarm appears, you will not be able to hear it for the first time

**Comment by Hobby4life** [comment]
Then there must be a extra Play Music option.

**Comment by miksterr** [comment]
Yes I agree, music is different to a call out, telemetry tr timer announcement which should always get played. Should music mute? duck? Or what during the call-out?

**Comment by chenbevis** [comment]
if  the sound is not played at the same time, it should at least resemble JETI Over write the previous sound instead of waiting for the previous sound to be played before continuing

**Comment by fullmetalpacket** [comment]
I also think Ethos is missing this feature badly.

**Comment by marciomc-github** [comment]
@bsongis-frsky is there any roadmap that we can see what are the enhancements prioritized to the upcoming versions?

**Comment by 3frcg** [comment]
Any news for Play Background Music ?

**Comment by DanPesonen** [comment]
Any News on Background Music?  I fly to music on my Horus and need it

**Comment by Radian327** [comment]
Any News on Background Music?  This would REALLY help with complex multiple timers.  I just have to trigger off a background music track that is an MP3 of my desired timer announcements...  This request has been logged for over two years and it would greatly help flyers.  Hope to see some process soon.

**Comment by crossup125** [comment]
While one can certainly fly without music and/or sounds, since the radio supports this in hardware its seems like a no brainer to use that capablity. I think without the ability to prioritize sounds, half the value in having them in the first place is lost.  Given the extreme depth of the programming ability we have with 1.5 it seems some more basic function like this is due, we are way past the point where a large percentage of users wont use a fraction of the capability for programming.

**Comment by Lenny970** [comment]
I also use long wav files to play custom timing tracks and would like to have the Play Background Music function in Ethos.

**Comment by camber2** [comment]
I also would like to see it back. I often listen to music while flying F5J.
I can easily get the file to play but can't stop it once started...

**Comment by chenbevis** [comment]
This problem should be easy to solve, But so far there has been no progress

**Comment by bsongis** [comment]
Perhaps because our TODO list is long?

**Comment by Lenny970** [comment]
Yes, the list is long and many new features have been added - some useful and some not.  It's just too bad that we've been waiting over 3 years now for the background music feature and it's still not available.  I guess it just hasn't been a priority.
